Sources/Defaults/Observation.swift

```diff
@@ -268,6 +363,23 @@ extension Defaults {
 		observation.start(options: options)
 		return observation
 	}
+	
+	public static func observe(
+		keys: DefaultsBaseKey...,
```

Comment:
I think https://github.com/sindresorhus/Defaults/issues/50 applies here too...

Reply:
If we added `DefaultsBaseKey` conformance to `Keys` and use it here, dot syntax would work. That would make `Keys` an abstract class and `DefaultsBaseKey` redundant.

Reply:
Alright. Let's do that.